### PR TITLE
Pull request template: simplify

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 Thank you for contributing!
 Please fill in the following checklist, removing items that do not apply.
 See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
+
+If you unsure whether to check something or not leave it unchecked.
 -->
 
-- [ ] The page (if new), does not already exist in the repository.
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
```
Comment (CleanMachine1) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: This makes sure that the user has some specification met. Removing it will cause more CI issues.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
+- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.).
 - [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
 - [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
 - [ ] The page description includes a link to documentation or a homepage (if applicable).
 
-**Version of the command being documented (if known):**
```
Comment (CleanMachine1) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: I like the current version

Comment (EmilySeville7cfg) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: I've almost reverted it:

```md
> :green_book: **Version of the command being documented:** `latest`
>
> Replace `latest` with a concreete version when you know it exactly.
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: I've reverted it.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
...
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+# Command info
 
-**Version of the command being documented (if known):**
+- **Version of the command being documented:** `latest`
+- **Brief description of the command being documented:** unspecified
+<!-- Replace `latest` with a concreete version when you know it exactly. -->
+
+- [x] New command pages or translations are added. <!-- Uncheck if you modify Tl;Dr tooling such as CI. -->
```
Comment (navarroaxel) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: Don't prefill values.
```suggestion
- [ ] New command pages or translations are added. <!-- Check if you create a documentation page to tldr-pages. -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
...
+- [ ] All command pages are in correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`. <!-- Check when you are sure in it. -->
```
Comment (navarroaxel) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: This comment really add information for the collaborator?
```suggestion
- [ ] All command pages are in correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
...
+- [ ] All command pages have 8 or fewer examples.
+- [ ] All command page descriptions have links to documentation or a homepage.
+- [ ] All command pages follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines). <!-- Check when you are sure in it. -->
```
Comment (navarroaxel) [inline .github/PULL_REQUEST_TEMPLATE.md:17]:
```suggestion
- [ ] All command pages follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines). <!-- Check when you are sure in it. -->
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (EmilySeville7cfg) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: I am not sure. 🤔

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
...
-- [ ] The page (if new), does not already exist in the repository.
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+# Command info
 
-**Version of the command being documented (if known):**
+- **Version of the command being documented:** `latest`
```
Comment (SethFalco) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: I don't think we should prepopulate answers in my opinion.

Especially as on Linux, it's not often that distributions will come with the latest version of any particular tool, so `latest` isn't a very good assumption/default either.

If we were to prepopulate it, I'd go for `unspecified`, but even then, I think leaving it blank is best.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
...
+- **Brief description of the command being documented:** unspecified
```
Comment (SethFalco) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
- **Summary of the command being documented:** unspecified
```

"Brief description" is a bit wordy.
(I'd also remove `unspecified`.)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
...
+<!-- Replace `latest` with a concreete version when you know it exactly. -->
```
Comment (SethFalco) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: I think this should be removed.
The form is intuitive as it is, there is no need to clarify this in my opinion.

It just wastes the time of the user who reads it.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
...
+
+- [ ] New command pages or translations are added. <!-- Check if you create a documentation page to tldr-pages. -->
```
Comment (SethFalco) [inline .github/PULL_REQUEST_TEMPLATE.md:13]:
```suggestion
- [ ] New command pages or translations are added. <!-- Check if you created a new page for tldr-pages. -->
```

I think the comment should be removed, but if we're going to keep it then a proposed improvement.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 10]
```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
...
+- [ ] All command pages are in correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.
```
Comment (SethFalco) [inline .github/PULL_REQUEST_TEMPLATE.md:14]:
```suggestion
- [ ] The page(s) are in the correct platform directory: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.
```

I'd move away from specifying "command" pages since we can have alias pages as well for example.

Also, I'd move away from specifying "all" since a contributor should only be responsible that the page they're trying to add is in the correct directory, not that all pages are.

Ideally, there shouldn't be too many new pages across directories anyway, they'd be split into multiple PRs.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 11]
```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
...
+- [ ] All command pages have 8 or fewer examples.
+- [ ] All command page descriptions have links to documentation or a homepage.
+- [ ] All command pages follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines). <!-- Check when you are sure in it. -->
+- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message). <!-- Check when you are sure in it. -->
```
Comment (SethFalco) [inline .github/PULL_REQUEST_TEMPLATE.md:18]:
```suggestion
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message). <!-- Check when you are sure in it. -->
```

I think the comment should be removed, it's implied that they should check. (It's the point of having it here, and why it's linked here.)

**Review by SethFalco — COMMENTED** [review]
Overall, I'm not actually that big a fan of this change myself.

Regardless, I've left some comments through it with my opinions.
Even if it's changed, I'd prefer what we had before. 🤔

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (CleanMachine1) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: Blank is definitely best, everything else is left for the user to fill, why make this one different

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 10 above)
Comment (EmilySeville7cfg) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: I consider alias pages as a sort of command pages just referring to another pages. 😄 We can implicitly mean `all` here. Or we can rewrite it as: `At least one page is in the correct platform directory: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.` But in this case user doesn't guarantees that he/she/they tried to correctly place all pages.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 9 above)
Comment (EmilySeville7cfg) [reply .github/PULL_REQUEST_TEMPLATE.md]: I've removed it.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 12]
```diff
@@ -4,11 +4,13 @@ Please fill in the following checklist, removing items that do not apply.
...
-- [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+# Command info
+
+- **Version of the command being documented:**  
 
-**Version of the command being documented (if known):**
+- [ ] New command pages or translations are added.
```
Comment (SethFalco) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: Also a query on this one.
PRs don't have to add new pages or translations, we sometimes have typos, poor formatting, localization issues, etc.

Comment (EmilySeville7cfg) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: I agree😁

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 10 above)
Comment (CleanMachine1) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: The Pages, suits better here.

The pages to me seems more like we are talking about the content being added 
whilst the 
All the new pages ... 
to me seems like a set of rules for all content, more like we are talking about the entire project rather than a users PR (I don't know how to explain it well, but thats the best I can up with ATM)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 13]
```diff
@@ -4,11 +4,13 @@ Please fill in the following checklist, removing items that do not apply.
...
 
-- [ ] The page (if new), does not already exist in the repository.
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+# Command info
+
+- **Version of the command being documented:**
```
Comment (sbrl) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: The (if known) is important here, as it may not be available.

Comment (EmilySeville7cfg) [reply .github/PULL_REQUEST_TEMPLATE.md]: Fixed. 😄

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 12 above)
Comment (SethFalco) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: Then maybe remove this one, or change it back to what we had before? ^-^'

People that want to update the README, update an existing page, or improve our documentation/templates don't have to add new pages.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 10 above)
Comment (SethFalco) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: Maybe we can go with "The pages" or "The page(s)" like before then, except plural?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 12 above)
Comment (EmilySeville7cfg) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: Then I think I have to remove `- [ ] All command pages have 8 or fewer examples.` because README for instance isn't a tl;dr page with just 8 examples.

Comment (EmilySeville7cfg) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: I like this idea. PR goal can be specified in it's title and there is no need to explicitly write about addition new pages or update of existing pages.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 10 above)
Comment (EmilySeville7cfg) [reply .github/PULL_REQUEST_TEMPLATE.md]: I've used `page(s)`.

Comment (sbrl) [reply .github/PULL_REQUEST_TEMPLATE.md]: I agree with @SethFalco on this.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 14]
```diff
@@ -6,11 +6,10 @@ See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
...
-- [ ] New command pages or translations are added.
-- [ ] All command pages are in correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.
-- [ ] All command pages have 8 or fewer examples.
+- [ ] All command page(s) are in correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.
+- [ ] All command page(s) have 8 or fewer examples.
 - [ ] All command page descriptions have links to documentation or a homepage.
-- [ ] All command pages follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
+- [ ] All command page(s) follow the [content guidelines](/tldr-page(s)/tldr/blob/main/CONTRIBUTING.md#guidelines).
+- [ ] The PR title conforms to the recommended [templates](/tldr-page(s)/tldr/blob/main/CONTRIBUTING.md#commit-message).
```
Comment (sbrl) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: These links are now broken?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 15]
```diff
@@ -4,11 +4,12 @@ Please fill in the following checklist, removing items that do not apply.
 See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
 -->
 
-- [ ] The page (if new), does not already exist in the repository.
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+# Command info
```
Comment (sbrl) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: I'm not sure having a header here is helpful.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 16]
```diff
@@ -4,11 +4,12 @@ Please fill in the following checklist, removing items that do not apply.
...
+
+- **Version of the command being documented (if known):**  
 
-**Version of the command being documented (if known):**
+- [ ] All command page(s) are in correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.
+- [ ] All command page(s) have 8 or fewer examples.
+- [ ] All command page descriptions have links to documentation or a homepage.
```
Comment (sbrl) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: Perhaps this will read better?
```suggestion
- [ ] All command page description(s) have links to documentation or a homepage.
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 15 above)
Comment (EmilySeville7cfg) [reply .github/PULL_REQUEST_TEMPLATE.md]: I've removed it.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 16 above)
Comment (EmilySeville7cfg) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yeah 😄

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 17]
```diff
@@ -4,11 +4,10 @@ Please fill in the following checklist, removing items that do not apply.
...
-- [ ] The page (if new), does not already exist in the repository.
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+- **Version of the command being documented (if known):**  
 
-**Version of the command being documented (if known):**
+- [ ] All command page(s) are in correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.
```
Comment (sbrl) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: This appears to be an exhaustive list without the `etc` at the end there, so we'll need to remember to update this when adding a new platform - which I'm not a fan of.

**Review by sbrl — APPROVED** [review]
This looks ok to me.

**Comment by sbrl** [comment]
Oh wait, you haven't addressed @SethFalco's comment @EmilySeville7cfg

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 18]
```diff
@@ -4,11 +4,10 @@ Please fill in the following checklist, removing items that do not apply.
...
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+- **Version of the command being documented (if known):**  
 
-**Version of the command being documented (if known):**
+- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
+- [ ] The page(s) have 8 or fewer examples.
```
Comment (marchersimon) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: I don't want to further delay this PR, so only apply this if you want.
```suggestion
- [ ] The page(s) have at most 8 examples.
```

**Review by marchersimon — APPROVED** [review]
